extensions/amp-cxense/0.1/amp-cxense.css

```diff
@@ -0,0 +1,76 @@
+/*
+    inspired from http://connoratherton.com/loaders
+*/
+.amp-cxense .amp-cxense-placeholder {
+    position: absolute;
+    top: 0;
+    height: 100%;
+    width: 100%;
+    overflow: hidden;
+    background: black;
+}
+.amp-cxense .amp-cxense-placeholder .amp-mpf-loader {
```

Comment:
would you happen to have an example usage somewhere? I'd be happy to reuse what's already built in